modules/setup-systemd-resolved/README.md

```diff
@@ -0,0 +1,64 @@
+# systemd-resolved Setup Script
+
+This folder contains a script for configuring [systemd-resolved](http://man7.org/linux/man-pages/man8/systemd-resolved.service.8.html) 
+to forward requests for a specific domain to Consul. This way, you can easily use Consul as your DNS server for
+domain names such as `foo.service.consul`, where `foo` is a service registered with Consul (see the [Registering 
+Services docs](https://www.consul.io/intro/getting-started/services.html) for instructions on registering your services
+with Consul). All other domain names will continue to be resolved via the default resolver on your OS. See the [Consul 
+DNS Forwarding Guide](https://www.consul.io/docs/guides/forwarding.html) and [Github Issue](https://github.com/hashicorp/consul/issues/4155) for more info.
```

Comment:
```suggestion
DNS Forwarding Guide](https://www.consul.io/docs/guides/forwarding.html) and [Github Issue](https://github.com/hashicorp/consul/issues/4155) for more info, including trade-offs between using this module and [dnsmasq](https://github.com/hashicorp/terraform-aws-consul/tree/master/modules/install-dnsmasq) for DNS forwarding.
```

(Could also add analogous sentence pointing back to this module in `install-dnsmasq` module)